.rubocop.yml

```diff
@@ -1,8 +1,11 @@
 AllCops:
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 3.1
   NewCops: enable
   SuggestExtensions: false
 
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
```

Comment:
[This cop](https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdevelopmentdependencies) was added, forcing all development dependencies out of the `gemspec` and into the `Gemfile`.  I looked at the discussion around the introduction of this cop, and wasn't convinced of the reasoning (when the repo is a gem): https://github.com/rubocop/rubocop/pull/11469

This rubocop config does the opposite, forcing all development dependencies to the `gemspec`, which is how we have it now.  I don't feel strongly either way, as long as all the dependencies are kept in the same place.  Happy to hear any arguments for using the cop as-is and putting the dependencies in the `Gemfile`!